### DISCUSSION TITLE
markdown not support checkbox

### DISCUSSION THREAD

**Discussion by bhabgs** [verdaccio/verdaccio#1393] [Ideas]
# markdown 

![image](https://user-images.githubusercontent.com/19221378/61271791-22323100-a7d8-11e9-9392-633e94e92235.png)

# 'verdaccio' readme show

![image](https://user-images.githubusercontent.com/19221378/61271849-4130c300-a7d8-11e9-9597-c76b96b63da6.png)

# 'github' readme show
![image](https://user-images.githubusercontent.com/19221378/61272012-a2f12d00-a7d8-11e9-9d52-f39d83f3ae7f.png)

**Comment by juanpicado** [comment]
hi @bhabgs we are limited to the https://github.com/markedjs/marked library. GitHub uses his own library as far I know, so, I'd rather research wether `marked` is able to handle that. If they cannot I'm afraid I cannot do anything about it.

**Comment by bhabgs** [comment]
this is on 'https://github.com/markedjs/marked'  
the same readme demo show..
![image](https://user-images.githubusercontent.com/19221378/61273879-1f860a80-a7dd-11e9-8a79-82c9e056a204.png)

**Comment by juanpicado** [comment]
https://github.com/markedjs/marked#usage yep, but they do not sanitize the HTML, we do for security reasons.

**Comment by juanpicado** [comment]
We have the parser here https://github.com/verdaccio/readme you can see all the scenarios we handle.  If you or anyone can actually make it work properly, I'll glad to support it.  PR welcome 👍

**Comment by bhabgs** [comment]
I would like to in my sapre time.

**Comment by yejinjian** [comment]
Are there any effective methods

**Comment by mbtools** [comment]
Fixed by https://github.com/verdaccio/verdaccio/pull/4494
